code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -60,7 +60,13 @@ func (sc *StorageSmartContract) addAllocation(alloc *StorageAllocation,
 			"saving new allocation in db: %v", err)
 	}
 
-	buff := alloc.Encode()
+	type Output struct {
+		ID      string `json:"id"`
+		Success bool   `json:"success"`
+	}
+
+	transaction_output := Output{alloc.ID, true}
+	buff, _ := json.Marshal(transaction_output)
 	return string(buff), nil
```

Comment:
Also, do we not have any Unit tests for this? It would be good to add UT for this atleast, so that it does not get changed and its reponse look different from others?

Reply:
updated the tests here https://github.com/0chain/0chain/pull/2542/files#diff-9c466ccb078e3cf5345cdda9ac180802725bc11fbbe4a944ffa1e99b61ed7e96R1246